### PR TITLE
Update remotecontrol layout

### PR DESCRIPTION
**Pull request by Nickbert7**
I have changed the layout of the remote control buttons (bigger on mobile devices) and moved them into a collapse

Old mobile
image

New mobile
image
image

Old desktop
image

New desktop
image

image

### CODE REVIEW

File: src/components/remotecontrol/remotecontrol.css [hunk 1]
```diff
@@ -12,6 +12,31 @@
     text-align: center
 }
 
+.btnArrowUp, .btnArrowLeft, .btnArrowRight, .btnArrowDown, .btnOk{
+    border: 2.7px solid rgba(255, 255, 255, .6);
+    background-color: rgba(0, 0, 0, .5);
+}
+
+.btnArrowUp{
+border-radius: 40% 40% 10% 10%;
```
Comment (JustAMan) [inline src/components/remotecontrol/remotecontrol.css]: I think we want indentation here and in similar places

File: src/components/remotecontrol/remotecontrol.js [hunk 2]
```diff
@@ -215,7 +215,7 @@ define(["browser", "datetime", "backdrop", "libraryBrowser", "listView", "imageL
                 context.querySelector(".sendTextSection").classList.add("hide");
             }
 
-            if (!currentPlayer.isLocalPlayer) {
+            if (-1 != supportedCommands.indexOf("MoveUp" || "MoveDown" || "MoveRight" || "MoveRight" || "MoveLeft" || "Select" || "Back" || "ToggleContextMenu" || "GoToSearch" || "GoToSettings" || "GoHome") && !currentPlayer.isLocalPlayer) {
```
Comment (JustAMan) [inline src/components/remotecontrol/remotecontrol.js]: are you sure this is doing what you think it does? :)
to me it seems you're actually only checking for `.indexOf("MoveUp")` ;)

**Comment by JustAMan** [comment]
Disagreed. We _already_ have those buttons, making them bigger on mobile is good for UX.

**Comment by dkanada** [comment]
It's not about making them bigger, it's the style itself.

File: src/components/remotecontrol/remotecontrol.js (hunk 2 above)
Comment (Nickbert7) [reply src/components/remotecontrol/remotecontrol.js]: Hm, yes it seems you're right... 😒
Do you have a hint for me how do solve it without checking for them seperate?
Or should I just keep the collapse visible with only disabled buttons?

I found out that I dont have to hide each button, they get disabled if not supported:
image

**Comment by Nickbert7** [comment]
I thought on getting them similar to the play button on the mobile layout.
image

**Comment by dkanada** [comment]
Ah, I was actually migrating the play buttons to a flat style. You can see the one I added on the media cards with no border and a nice transition effect.

**Comment by Nickbert7** [comment]
Ok, I'll have a look on it tomorrow or next week.
Maybe I should also check some other remote control apps to get some impressions.

**Comment by Nickbert7** [comment]
I have removed the border and use the color of the submit button now
image

**Pull request merged**